lib/developer_portal/app/controllers/developer_portal/admin/account/personal_details_controller.rb

```diff
@@ -19,7 +19,9 @@ def show
   def update
     resource.validate_fields!
     if resource.errors.empty? && resource.update(user_params)
-      resource.kill_user_sessions(user_session) if resource.just_changed_password?
+      if resource.just_changed_password?
+        resource.kill_user_sessions(user_session)
+      end
```

Comment:
why this change? Seems like rubocop prefers single line :)

Reply:
But actually wasn't `password_digest` attr important now?

Reply:
> why this change? Seems like rubocop prefers single line :)

Yeah, it was single line before but I made it complete form because I was also adding the audit log here. But then I removed the audit log and forgot to make it single line again.

> But actually wasn't password_digest attr important now?

`password` is a field in the form, not the DB column, that is called `password_digest` since we don't save passwords in plain text. In fact, look how is `just_changed_password` implemented:

https://github.com/3scale/porta/blob/843f799f292f34cb6e26fa5e9ab2a34526ef2ae3/app/lib/authentication/by_password.rb#L41-L43

Reply:
Fixed the `if`: https://github.com/3scale/porta/pull/3885/commits/df2fa1986850eca6f32cb5720921b13c14f2d67d